packages/tools/generator-widget/README.md

```diff
@@ -94,28 +73,44 @@ It has the following features:
 
 The empty template is a Mendix React hello world widget recommended for more experienced developers.
 
-### 2.3 Add unit tests for the widget ?
+### Add unit tests
 
 If `Yes` is selected, unit tests are included to ensure individual units of the component are tested to determine whether they are fit for use. Default value is `No`.
 
-### 2.4 Add end to end tests for the widget ?
+### Add end to end tests
 
-If Yes is selected, end to end tests are included to ensure that the integrated components of an application function as expected. Default value is `No`.
+If `Yes` is selected, end to end tests are included to ensure that the integrated components of an application function as expected. Default value is `No`.
 
 Note: Both `Unit` and `End to end` tests apply only to the Full Boilerplate. `End to End` is exclusive for web and hybrid mobile apps.
 
 The tool will then create copied files, and run `npm install` to install development dependencies.
 
-### NOTE
+## Using the task runner
 
-To use the webpack-dev-server while developing:
+The widget generator will include the necessary files and tasks to your package.json for running the tasks over the [Pluggable Widgets Tools](https://github.com/mendix/widgets-resources/tree/master/packages/tools/pluggable-widgets-tools).
 
--   Start Mendix Studio Pro from your Mendix project path or by default `/dist/MxTestProject` and run:
+If necessary you can run the tasks using the commands:
 
 ```bash
 npm start
 ```
 
+```bash
+npm run build
+```
+
+```bash
+npm run release
+```
+
+## Note
+
+-   To use the webpack-dev-server while developing, run the Mendix project located at the specified `Mendix project path` and run:
```

Comment:
If someone doesn't know `webpack-dev-server`, why would they want to run this command? Maybe mention something about _what_ the dev server does; "To build and watch for source-code changes..."

Reply:
Good point! Changed it in the readme.